### DISCUSSION TITLE
PrimeFaces 13.0 and aristo style

### DISCUSSION THREAD

**Discussion by xidbhonaker** [primefaces/community#224] [PrimeFaces] [answered]
Has anyone made a 13.0-compatible theme that is similar to the aristo theme that was part of the base themes in earlier releases?  

Or can point me to steps to do that myself?

My older app looks different, and I'd like to keep the users happy (and as unaware as possible about the upgrade behind the curtain).

**Comment by NicolaIsotta** [comment, marked as answer]
I don't think someone has recreated Aristo, given its outdated appearance, but you can do it:

- https://www.primefaces.org/designer-jsf
- https://github.com/primefaces/primefaces-sass-theme

**Reply by melloware** [reply]
@NicolaIsotta answer is the correct answer.  You would need to make a new theme that looks like Aristo.

**Comment by xidbhonaker** [comment]
@NicolaIsotta, I thought that might be the case.  However, the 2 links you sent have no 'users guide'.  Has anyone created a tutorial on how to use these?  Either text- or video- based would be great.

Having access to this project is certainly a requirement, but are not a good starting point.

**Comment by NicolaIsotta** [comment]
General documentation abouth theming: https://primefaces.github.io/primefaces/13_0_0/#/core/themes
The visual designer is WISIWYG so pretty self-explanatory.
For the sass theme the docs are here: https://www.primefaces.org/designer/api/primefaces/13.0.0/

**Comment by xidbhonaker** [comment]
Thanks, Nicola,

That's a start although once upon a time, I might have seen a higher level starter document describing basic steps.  I'll start down this path next week.

I really appreciate your pointers and experience.
Bill

**Reply by NicolaIsotta** [reply]
This video may help: https://youtu.be/juP3xkE8mvI

**Reply by xidbhonaker** [reply]
Thanks again, Nicola.  I'll watch that.  First few minutes looked promising.

**Reply by melloware** [reply]
I just uploaded the last Aristo to Maven Central.

```xml
<dependency>
    <groupId>org.primefaces.themes</groupId>
    <artifactId>aristo</artifactId>
    <version>1.1.0</version>
</dependency>
```
